profiling/tests/correctness/time.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
As discussed, structuring the scenarios so they can be fetched would be nice.
I would suggest:
- correctness/{scenario}/expected.json
- correctness/{scenario}/main.php (or whatever can be used as a standard entry point ?)

Reply:
I would like to do this when we are working on implementing the counter part in https://github.com/DataDog/prof-correctness

Reply:
Sorry I was not clear maybe, I just was suggesting adding an extra directory after correctness. 
Though I agree we can wait for the counter part in prof-correctness.
Generally I still think prof-correctness will eventually contain a number of different images on which you should run the test. Then you would give it a script to run, and an expected. That would run on the prof-correctness repository.

I'm fine with adjusting when needed :-)